From bd52755f73b3a58e5fffc640d7ab8ae1045787fc Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Tue, 16 Feb 2021 11:31:00 +0800
Subject: [PATCH] ARM: dts: rockchip: make the empty node of gmac not empty for
 rv1126-thunder-boot

The significant nodes will be deleted with CONFIG_DTC_OMIT_EMPTY.
therefore make it not empty.

Change-Id: Id5febc54e9efe2d769a30b6ded96288a37239e90
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-thunder-boot.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-thunder-boot.dtsi b/arch/arm/boot/dts/rv1126-thunder-boot.dtsi
index 7eb994b2352b..5115b360c9fd 100644
--- a/arch/arm/boot/dts/rv1126-thunder-boot.dtsi
+++ b/arch/arm/boot/dts/rv1126-thunder-boot.dtsi
@@ -101,6 +101,18 @@
 	inactive;
 };
 
+&mtl_rx_setup {
+	queue0 {
+		status = "okay";
+	};
+};
+
+&mtl_tx_setup {
+	queue0 {
+		status = "okay";
+	};
+};
+
 &ramoops {
 	status = "disabled";
 };
-- 
2.35.3

